packages/front-end/pages/settings/index.tsx

```diff
@@ -987,21 +992,30 @@ const GeneralSettingsPage = (): React.ReactElement => {
               <div className="col-sm-9">
                 <div className="form-inline">
                   <Field
-                    label="Amount of historical data to include when analyzing metrics"
+                    label="Amount of historical data to use on metric analysis page"
+                    type="number"
                     append="days"
                     className="ml-2"
-                    containerClassName="mb-3"
+                    containerClassName="mb-0"
                     disabled={hasFileConfig()}
-                    options={[7, 14, 30, 90, 180, 365]}
                     {...form.register("metricAnalysisDays", {
                       valueAsNumber: true,
                     })}
                   />
+                  {metricAnalysisDaysWarningMsg && (
+                    <small
+                      style={{
+                        color: "#e27202",
```

Comment:
Don't hardcode colors.  Use class names instead.  `<small className="text-danger">` for red.  `<small className="text-warning">` for yellow (can be hard to read sometimes).

Reply:
Ok, there's a lot of hard coded colors in this file, fwiw. I'll go to text-danger since that looks fine.